src/replicate-incrementally.js

```diff
@@ -51,36 +54,50 @@ async function run(configuration) {
       lastRecordIndex: answersLastRecordIndexTargetBeforeReplication,
     },
     {
-      name: '\\"knowledge-elements\\"',
+      name: 'knowledge-elements',
       lastRecordIndex: kELastRecordIndexTargetBeforeReplication,
     },
     {
-      name: '\\"knowledge-element-snapshots\\"',
+      name: 'knowledge-element-snapshots',
       lastRecordIndex: kESnapshotsLastRecordIndexTargetBeforeReplication,
     },
   ];
 
-  tablesAndLastRecordIndex.forEach((table) => {
-    const sqlCopyCommand = `
-      psql \\
-          ${configuration.SOURCE_DATABASE_URL} \\
-          --command "\\copy (SELECT * FROM ${table.name} WHERE id>${table.lastRecordIndex}) to stdout" | \\
-      psql \\
-          ${configuration.TARGET_DATABASE_URL} \\
-          --command "\\copy ${table.name} from stdin"`;
-
-    const copyMessage = execSync(sqlCopyCommand);
-    logger.info(`${table.name} table copy returned: ` + copyMessage);
-  });
-
-  tablesAndLastRecordIndex.forEach(async(table) => {
-
-    const maxIdStrAfterReplication = await execStdOut('psql', [configuration.TARGET_DATABASE_URL, '--tuples-only', '--command', `SELECT MAX(id) FROM ${table.name.replace(/\\/g, '')}`]);
+  for (const table of tablesAndLastRecordIndex) {
```

Comment:
Pourquoi `for` plutôt que `await Promise.all( tablesAndLastRecordIndex.map({..})) )` ?
Pour éviter les copies en // et ainsi ne pas trop solliciter la base ?

Reply:
Exactement !

Reply:
Humm... pourrait-on mieux exprimer cette intention ?
Extraire dans une méthode `copyMissingDataSeriallyByTable` par exemple ?

Reply:
Bof, si on va par là j'ai l'impression qu'on pourrait l'appliquer à toutes les boucles `for` :sweater